ludwig/api.py

```diff
@@ -1788,6 +1793,9 @@ def load(
         :param callbacks: (list, default: `None`) a list of
             `ludwig.callbacks.Callback` objects that provide hooks into the
             Ludwig pipeline.
+        :param from_checkpoint: (bool, default: `False`) if `True`, the model
+            will be loaded from the latest checkpoint (training_checkpoints/)
+            instead of the final model weights.
```

Comment:
Yeah I guess this is a fair thing to do, wondering though what the reason would be to load from_checkpoint if `model/model_weights` is already present? Perhaps a no-op in that case and we can always make this True? Okay keeping it like this for now as well, just wanted to call it out

Reply:
Yeah– I think it's okay to have it be explicit and not make assumptions on behalf of the user for now. 

We can revisit this if people get confused, but because it's default False, I'm hopeful this won't interrupt anyone's experience (until of course someone really needs it, at which point we can direct them to this flag)